index.html

```diff
@@ -0,0 +1,484 @@
+            </table>
+            <table class="l-flex-item l-flex-prices">
+              <thead>
+                <tr>
+                  <th class="f-light th-title">STANDART</th>
+                </tr></thead>
+              <tbody>
+                <tr>
+                  <td class="f-bold f-client">$10.99<span class="f-price"> / Month</span></td>
+                </tr>
+                <tr>
+                  <td>7 Projects</td>
+                </tr>
+                <tr>
+                  <td>5 Users</td>
+                </tr>
+                <tr>
+                  <td>10 Gb user</td>
+                </tr>
+                <tr>
+                  <td>5 Calendar</td>
+                </tr>
+              </tbody>
+              <tfoot>
+                <tr>
+                  <td class="td-btn"><a href=""><button class="btn f-bold">SIGN UP</button></a></td>
+                </tr>
+              </tfoot>
+            </table>
+            <table class="l-flex-item l-flex-prices">
+              <thead>
+                <tr>
+                  <th class="f-light th-title">PROFESSIONAL</th>
+                </tr></thead>
+              <tbody>
+                <tr>
+                  <td class="f-bold f-client">$25.99<span class="f-price"> / Month</span></td>
+                </tr>
+                <tr>
+                  <td>15 Projects</td>
+                </tr>
+                <tr>
+                  <td>50 Users</td>
+                </tr>
+                <tr>
+                  <td>100 Gb user</td>
+                </tr>
+                <tr>
+                  <td>25 Calendar</td>
+                </tr>
+              </tbody>
+              <tfoot>
+                <tr>
+                  <td class="td-btn"><a href=""><button class="btn f-bold">SIGN UP</button></a></td>
+                </tr>
+              </tfoot>
+            </table>
+            <table class="l-flex-item l-flex-prices">
+              <thead>
+                <tr>
+                  <th class="f-light th-title">PREMIUM</th>
+                </tr></thead>
+              <tbody>
+                <tr>
+                  <td class="f-bold f-client">$49.99<span class="f-price"> / Month</span></td>
+                </tr>
+                <tr>
+                  <td>UNLIMITED Projects</td>
+                </tr>
+                <tr>
+                  <td>UNLIMITED Users</td>
+                </tr>
+                <tr>
+                  <td>1000 Gb user</td>
+                </tr>
+                <tr>
+                  <td>UNLIMITED Calendar</td>
+                </tr>
+              </tbody>
+              <tfoot>
+                <tr>
+                  <td class="td-btn"><a href=""><button class="btn f-bold">SIGN UP</button></a></td>
+                </tr>
+              </tfoot>
+            </table>
+          </div>
+        </section>
+      </article>
+      <article class="b-white" id="portfolio">
+        <h2 class="f-bold f-large">PORTFOLIO</h2>
```

Comment:
The lines to either side of this are broken in the smaller mobile (320x568) dimension.